README.rst

```diff
@@ -597,15 +602,12 @@ The ``psr2`` level is set by default, you can also change the default level:
 
 In combination with these config and command line options, you can choose various usage.
 
-For example, default level is ``psr2``, but if you also don't want to use
+For example, the default level is ``psr2``, but if you don't want to use
 the ``psr0`` fixer, you can specify the ``--fixers="-psr0"`` option.
 
 But if you use the ``--fixers`` option with only exact fixers,
 only those exact fixers are enabled whether or not level is set.
 
-With the ``--config-file`` option you can specify the path to the
-``.php_cs`` file.
-
 By using ``--using-cache`` option you can set if caching
```

Comment:
My I suggest
`By using the ``--using-cache`` option with ``yes`` or ``no`` you
+can set if the caching mechanism should be used. `
(I think this was why I started the previous pr's ;)  )